contracts/testHelpers/VotingReputationMisalignedFixed.sol

```diff
@@ -0,0 +1,1021 @@
+/*
```

Comment:
This looks like changes related to a separate issue -- does this belong in this PR?

Reply:
These changes are to fix the tests around the storage misalignment issue, which are necessary because the version for VotingReputation is bumped here.

These could be in another PR, or indeed removed entirely (do we need tests showing that upgrade works, now it's in the past?)

Reply:
I would be happy to remove this then, it's a lot of code that serves minimal purpose.